modules/core/src/main/java/org/apache/ignite/configuration/ClientConnectorConfiguration.java

```diff
@@ -349,6 +357,34 @@ public ClientConnectorConfiguration setSelectorCount(int selectorCnt) {
         return this;
     }
 
+    /**
+     * Gets maximum allowed number of active client connections per node.
+     * <p>
+     * Defaults to {@link #DFLT_MAX_CONNECTION_CNT}.
+     * This applies to any connections that use thin client protocol: thin clients, ODBC, thin JDBC connections.
```

Comment:
The limit is shared, right? Limit of 10 means `thin+ODBC+JDBC <= 10`?

Reply:
Yep. Clarified.